File: 2022/ISeleb/app/src/main/java/com/creator/iseleb/util/ProgressUtil.java
package com.creator.iseleb.util;

import android.app.Dialog;
import android.content.Context;
import android.view.Window;

import com.creator.iseleb.R;

/**
 * Created by skcrackers on 5/27/16.
 */
public class ProgressUtil extends Dialog {
    public ProgressUtil(Context context) {
        super(context);
        requestWindowFeature(Window.FEATURE_NO_TITLE);
        setContentView(R.layout.view_progress_util);
    }
}

File: 2025/UnniTv/app/src/main/java/com/creator/unnitv/util/ProgressUtil.kt
package com.creator.unnitv.util

import android.app.Dialog
import android.content.Context
import android.view.Window
import com.creator.unnitv.R

/**
 * Created by skcrackers on 5/27/16.
 */
class ProgressUtil(context: Context?) : Dialog(context!!) {
    init {
        requestWindowFeature(Window.FEATURE_NO_TITLE)
        setContentView(R.layout.view_progress_util)
    }
}